backend/ee/danswer/db/analytics.py

```diff
@@ -170,3 +170,67 @@ def fetch_danswerbot_analytics(
     )
 
     return results
+
+
+def fetch_persona_message_analytics(
+    db_session: Session,
+    persona_id: int,
```

Comment:
Probably worth combining shared logic across the functions here

Reply:
tried it. its ugly, confusing, and probs not worth